manage/manageShips.php

```diff
@@ -1,20 +1,20 @@
 <?php
```

Comment:
I think we should add another subdirectory within `manage` called `ships`. Then we can rename this file `index` since it is in the ships directory. This will allow you some flexibility as more things get added into the management side of things.

The header and footer files can stay in the manage directory though

Reply:
Also created a manage/layout folder along with the manage/ships folder. I updated the layout folder for my sanity when locating the files and passing the bootstrap information in. Updated routes as needed.

Reply:
I think the name of this file should change to `index.php` too.